.github/workflows/sync-next-branch-with-smithy-rs.yaml

```diff
@@ -0,0 +1,58 @@
+on: workflow_dispatch
+
+env:
+  rust_version: 1.54.0
+  java_version: 11
+
+name: Sync `next` branch with smithy-rs
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: Sync with smithy-rs
+    steps:
+      - name: Clone smithy-rs
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-labs/smithy-rs'
+      - uses: actions/cache@v2
+        name: Gradle Cache
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+        # Pinned to the commit hash of v1.3.0
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.java_version }}
+      - name: Generate new SDK
+        working-directory: smithy-rs/
+        run: |
+          ./gradlew rust-runtime:assemble
```

Comment:
I don't think this generates the full SDK? I think we need aws:sdk:assemble, right?

Reply:
Good catch. This should be:
```
 ./gradlew -Paws.fullsdk=true :aws:sdk:assemble
```
I don't think the `rust-runtime:assemble` step is necessary since `aws:sdk:assemble` repeats that work, but double check by looking at the version numbers and dependencies in the output runtime crates.